java/jmh/README.md

```diff
@@ -0,0 +1,18 @@
+# JMH Benchmarks for RocksJava
+
+These are micro-benchmarks for RocksJava functionality, using [JMH](Java Microbenchmark Harness - https://openjdk.java.net/projects/code-tools/jmh/).
```

Comment:
Nit: malformed markdown link

Reply:
I see you have imported it to Phabricator. Should I still push an update to fix the markdown?

Reply:
I guess I'm sending mixed signals. Sorry. Yes, please update.

Phabricator import is required for our internal validation, and that's more useful to run early enough to share results with the author, saving round trips. We (or at least I) treat the PR as source of truth until closed, but it probably makes sense to assume we might be trying to land the PR if it's accepted and imported. So I'll try to be decisive about accepted vs. request changes.

For this PR, internal checks saw that LICENSE-HEADER.txt is missing a trailing newline. Please fix.

If you generally don't mind me pushing cosmetic things to your PR branches, I can probably fix them that way faster than round-trip. Let me know.

Reply:
Thanks @pdillinger. I just fixed those two small things and pushed.
I don't mind you pushing cosmetic commits at all, after all open-source is a team sport ;-)